bus-mapping/src/evm/opcodes/create.rs

```diff
@@ -251,7 +251,6 @@ impl<const IS_CREATE2: bool> Opcode for Create<IS_CREATE2> {
                     (CallContextField::IsStatic, false.to_word()),
                     (CallContextField::IsCreate, true.to_word()),
                     (CallContextField::CodeHash, code_hash.to_word()),
-                    (CallContextField::Value, callee.value),
```

Comment:
Does it means this lookup is not using by circuit originally?

Reply:
yes, we don't have this constraint this in our circuit.